### DISCUSSION TITLE
Dsc alexor or scw9057-433mhz

### DISCUSSION THREAD

**Discussion by rdaguerre** [taligentx/dscKeybusInterface#163] [dscKeybusInterface Discussion]
Hi, where can i find a mqtt/IoT solution for dsc wireless systems?
Thanks for all

Ronnie

**Comment by taligentx** [comment]
Hi Ronnie, I'm not aware of a way to connect to the DSC wireless systems, there's no direct connection to the Keybus for this interface to wire into.  You'd need to use a PowerSeries panel with a RF keypad or the 5132 wireless module to use this interface with wireless sensors and IoT connectivity.
